core/lib/sdk.ts

```diff
@@ -996,9 +998,11 @@ notes.push(tmpNote);
      * get all transactions
      *
      * @param {Object} ctx
+     * @param {Object} page
+     * @param {Object} page_size
      * @return {Object} transactions
      */
-    async getTransactions(ctx: any) {
-        return this.trans.getTransactions(ctx);
+    async getTransactions(ctx: any, page: any, page_size: any) {
```

Comment:
put all the paging arguments into a single argument, option, where option = { page: page, pageSize: pageSize}.